src/NuGet.Services.AzureSearch/DownloadTransferrer.cs

```diff
@@ -23,9 +33,25 @@ public DownloadTransferrer(ILogger<DownloadTransferrer> logger)
             SortedDictionary<string, SortedSet<string>> outgoingTransfers,
             IReadOnlyDictionary<string, long> downloadOverrides)
         {
-            // TODO: Add download changes due to popularity transfers.
-            // See: https://github.com/NuGet/NuGetGallery/issues/7898
-            var downloadTransfers = new SortedDictionary<string, long>(StringComparer.OrdinalIgnoreCase);
+            Guard.Assert(
```

Comment:
Should we have null-check here considering the public methods?

Reply:
We generally don't add null checks to our public methods.